### ISSUE TITLE
How are JavaScript and jQuery different?

### ISSUE THREAD

**Issue by bofeiw** [shamsmosowi/git-bot#30]

**Comment by shamsmosowi** [comment]
Hi,

JavaScript and jQuery are both programming languages used to develop websites and web applications. The main difference between them is that JavaScript is a scripting language, while jQuery is a library built on JavaScript. 

JavaScript is a programming language used to create dynamic webpages and applications, while jQuery is a library of JavaScript functions that make it easier to use JavaScript to create dynamic webpages and applications. JavaScript can be used to create interactive webpages, while jQuery simplifies web development by providing a set of pre-written functions to interact with webpages. 

In short, JavaScript is a scripting language while jQuery is a library of functions to simplify web development. 

I hope this helps!
